godel_scan_analysis/src/godel_scan_test_publisher.cpp

```diff
@@ -0,0 +1,32 @@
+#include <ros/ros.h>
+
+#include <pcl_ros/point_cloud.h>
+#include <pcl/point_types.h>
+
+typedef pcl::PointCloud<pcl::PointXYZ> point_cloud_t;
+
+void generateFakeData(point_cloud_t& cloud)
+{
+  static double x = 0.0;
+  static double y = 0.0;
+  static double z = 0.0;
+  cloud.points.push_back(pcl::PointXYZ(x, y, z));
```

Comment:
The hope was to, at some point, provide more advanced features. Since this file doesn't really serve a purpose at the moment, I'm just going to delete it all together.